app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
```

Comment:
provide additional typehint to what is in the list

Reply:
![image](https://user-images.githubusercontent.com/116162917/219423592-8663f0f4-b344-4fc7-9a06-f4573600a86f.png)
Can you tell me how to fix this error as I couldn't find anything online